docs/data/sdks/ios-swift/index.md

```diff
@@ -0,0 +1,484 @@
+---
+title: iOS SDK
```

Comment:
Add (Beta) in title?

Reply:
Thanks for reviewing! Good catch, updated in latest commit.